db/seeds/development.rb

```diff
@@ -0,0 +1,17 @@
+
+n = rand(1..2000)
+12.times do
+  Bar.create!(name: "Mayos Frozen Bar #{n}",
+              address: "234 That Street #{n}",
+              city: "Boston",
+              state: "Massachusetts",
+              zip: "02345",
+              description: "You won't believe it. You just won't",
+              url: "http://thatsite.com",
+              photo_url: "http://thatothersite.com",
+              seating_capacity: 4,
+              food: true,
+              outdoor_seating: false,
+              pet_friendly: true)
+  n += 1
+end
```

Comment:
Just because you'll be using it often, I would suggest finding real bar data and populating the seed with that. It isn't necessary right now, but it will make the site look nicer later on.